interface-definitions/nat64.xml.in

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="nat64" owner="${vyos_conf_scripts_dir}/nat64.py">
+    <properties>
+      <help>IPv6-to-IPv4 Network Address Translation (NAT64) Settings</help>
+      <priority>500</priority>
```

Comment:
Please use 501 here as 500 is used by nat66 - only to get a deterministic behavior.